tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            self.mdg, self.box = pp.md_grids_2d.two_intersecting(self.params.get("mesh_args", None),
</pre><p> </p><p>These are the cases where I wish everything to fit into a line. But this is purely esthetics.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>